projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -2151,16 +2152,16 @@ private void serializeConversionContext(
         cpServerData = readAllInputObjects(keys, snapshot);
       }
       if (!cpServerData.isEmpty()) {
+        LOGGER.info("\n*** READING CHECKPOINT MANAGEMENT CONFIGS ***\n");
```

Comment:
eh not thrilled about this change actually, removes I/O from log.

Reply:
Mitigated by separating listing keys and reading data. The listing is fast, so it's "okay" that's hidden in logs IMO.